config/m4/lcov.m4

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+#
+# Enable LCOV capable build
+#
+
+AC_ARG_ENABLE([lcov],
+    AS_HELP_STRING([--enable-lcov], [Enable code coverage reporting]),
+    [],
+    [enable_lcov=no])
+
+lcov_happy=no
+AS_IF([test "x$enable_lcov" = xno],
+    [AC_MSG_WARN([LCOV support is not enabled.])],
+    [
+        AC_CHECK_PROG([GCOVBIN], gcov, yes)
```

Comment:
IMO better use other string (not yes) in case it's not found
maybe empty string or "notfound"

Reply:
agree, fixed